test/napi/lib/objects.js

```diff
@@ -28,6 +28,33 @@ describe("JsObject", function () {
     );
   });
 
+  it("freeze a JsObject", function () {
```

Comment:
Can you add a test for freezing something that can't be frozen, like an Array view?

Reply:
I can't seem to find any documentation on an `ArrayView`, and freezing arrays as well as array buffers works fine. Do you mind pointing me in the right direction here?

Reply:
[`ArrayBufferView`](https://developer.mozilla.org/en-US/docs/Web/API/ArrayBufferView) is the name for typed arrays like `Uint32Array`.

```
$ node -e 'Object.freeze(new Uint32Array(8))'
[eval]:1
Object.freeze(new Uint32Array(8))
       ^

TypeError: Cannot freeze array buffer views with elements
    at Function.freeze (<anonymous>)
    at [eval]:1:8
    at Script.runInThisContext (node:vm:129:12)
    at Object.runInThisContext (node:vm:305:38)
    at node:internal/process/execution:75:19
    at [eval]-wrapper:6:22
    at evalScript (node:internal/process/execution:74:60)
    at node:internal/main/eval_string:27:3
```